zipp.py

```diff
@@ -61,6 +61,39 @@ def _difference(minuend, subtrahend):
     """
     return itertools.filterfalse(set(subtrahend).__contains__, minuend)
 
+class PickleableClass():
```

Comment:
The naming of this class expresses the outcome more than its essential purpose (to defer initialization). I'll probably rename it to something like `DeferredInitialization`.

Reply:
I've renamed it to `InitializedState`.